setup.cfg

```diff
@@ -1,9 +1,9 @@
 [metadata]
 # replace with your username:
 name = epython
-version = 0.0.1
+version = "2021.1002"
```

Comment:
Given this is a rolling release, this version scheme seemed reasonable to give users an idea of how long ago it was updated just by looking at the version.